### PR TITLE
2023 SC election results

### PR DESCRIPTION
**Pull request by ReToCode**
# Changes
- As suggested in https://github.com/knative/community/tree/main/elections#process, publishing 2023 SC election results, so that it shows up in Elekto properly.

/assign @geekygirldawn

### CODE REVIEW

File: elections/2023-SC/results.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+## 2023 SC Member Election Results
+
+We are happy to announce the results of the 2023 Knative Steering Committee (SC) election results!
+
+Please welcome Ali Ok and Evan Anderson as new SC member. They will serve a 2 years term ending in 2025 starting immediately.
```
Comment (geekygirldawn) [inline elections/2023-SC/results.md:5]:
```suggestion
Please welcome Ali Ok and Evan Anderson as new SC members. They will serve a 2 year term ending in 2025 starting immediately.
```

File: elections/2023-SC/results.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+We would also like to extend our deepest gratitude to the SC members stepping down for their outstanding service:
+
+Lance Ball and Carlos Santana, who are both ending their two years term.
```
Comment (geekygirldawn) [inline elections/2023-SC/results.md:9]:
```suggestion
Lance Ball and Carlos Santana, who are both ending their terms.
```

File: elections/2023-SC/results.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+Cheers,
+Dawn Foster and Reto Lehmann
+TOC Election Officers
```
Comment (geekygirldawn) [inline elections/2023-SC/results.md:13]:
```suggestion
SC Election Officers
```

**Review by geekygirldawn — COMMENTED** [review]
I made a couple of quick suggestions to the wording.

**Comment by ReToCode** [comment]
> I made a couple of quick suggestions to the wording.

So much for believing the IDEs grammar skills 😅

**Review by geekygirldawn — APPROVED** [review]

**Pull request merged**